AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/linked-participant.model.ts

```diff
@@ -0,0 +1,12 @@
+export class LinkedParticipantModel {
```

Comment:
All of these fields are optional (?) so you don't need to use the union type `string | undefined` - I would just type them as strings - the compiler will allow them to be undefined because they are optional

Reply:
updated to string